frontend/src/app/main-landing/main-landing.component.html

```diff
@@ -44,32 +44,34 @@ <h3 class="card-title">Temporary Outfitting and Guiding Permit</h3>
   </div>
 
 
-  <div class="help-pick-flex" id="help-pick-background">
-    <div class="small-col small-col-left">
-      <span class="hero-icon">
-        <i class="far fa-question-circle usa-ico"></i>
-      </span>
+  <div id="help-pick-background">
+    <div class="help-pick-flex">
+      <div class="small-col small-col-left">
+        <span class="hero-icon">
+          <i class="far fa-question-circle usa-ico"></i>
+        </span>
+      </div>
+      <div class="big-col">
+        <h2 class="left-align">I’m not sure which permit I need...</h2>
+        <p class="left-align">If you’re not sure about what kind of permit you need, click on the button below to answer a few questions. We will guide you to the permit that best meets your needs.</p>
+      </div>
+      <div class="small-col">
+        <a id="help-find-permit" class="usa-button-primary-alt usa-button usa-button display-inline" routerLink="/special-use/help-me-pick/1">Help me find a permit</a>
+      </div>
     </div>
-    <div class="big-col">
-      <h2 class="left-align">I’m not sure which permit I need...</h2>
-      <p class="left-align">If you’re not sure about what kind of permit you need, click on the button below to answer a few questions. We will guide you to the permit that best meets your needs.</p>
-    </div>
-    <div class="small-col">
-      <a id="help-find-permit" class="usa-button-primary-alt usa-button usa-button display-inline" routerLink="/special-use/help-me-pick/1">Help me find a permit</a>
-    </div>
-  </div>
 
-  <div class="help-pick-flex" id="help-pick-background">
-    <div class="small-col small-col-left">
-      <span class="hero-icon">
-        <i class="fas fa-tree usa-ico"></i>
-      </span>
-    </div>
-    <div class="big-col">
-      <h2 class="left-align">Looking for Christmas trees?</h2>
-      <p class="left-align">Please click on the following link to be redirected to the new online sales of Christmas Tree permits for the USFS at <a href="https://www.recreation.gov/">recreation.gov</a></p>
```

Comment:
Not sure if this block is scope creep or not. I don't have any work in progress on code, but it doesn't seem related to 508.